editor/components/block-switcher/index.js

```diff
@@ -9,7 +9,7 @@ import clickOutside from 'react-click-outside';
  * Internal dependencies
  */
 import './style.scss';
-import IconButton from 'components/icon-button';
+import IconButton from '../icon-button';
 
 class BlockSwitcher extends wp.element.Component {
```

Comment:
>Should we change these to `import { Component } from 'element';`?

Yep, I'll cover this in this next step: "Replace `wp.` global references with respective "WordPress dependency" equivalent"